### PR TITLE
Add --disable_watches flag

### PR DESCRIPTION
**Pull request by miekg**
Setting this flag to true makes fleet not set any watches in etcd.

### CODE REVIEW

**Comment by miekg** [comment]
Ah, yeah.
1. Haven't added prometheus dependency in Godeps
2. `./test` works. Hasn't seen any production use.

**Comment by jonboulle** [comment]
@miekg we'd like to clean this up and get it merged. What's the reason you removed the jitter stuff?

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [fleet] Add --disable_watches f..." ]
>@miekg we'd like to clean this up and get it merged. What's the reason you removed the jitter stuff?

failing tests... I couldn't (quickly) find wth was happening there.

**Comment by jonboulle** [comment]
Hmm, ok - do you want to squash these commits and we can land as-is then?

**Comment by jonboulle** [comment]
and tweak the commit message :-)

**Comment by miekg** [comment]
done.

File: Documentation/fleet-scaling.md [hunk 1]
```diff
@@ -55,3 +55,9 @@ wins:
 * Making some defaults exported and allow them to be overridden. For instance
     fleet's tokenLimit controls how many Units are listed per "page". *See the
     `--token-limit` flag.*
+
```
Comment (jonboulle) [inline Documentation/fleet-scaling.md]: Er, sorry. I thought I commented here but apparently not. Can you remove lines 36-46 from this file?

**Comment by miekg** [comment]
lines 36-46 removed.

**Comment by jonboulle** [comment]
@miekg thanks for your patience :-)

**Comment by anwald** [comment]
If changes to schedule state in etcd are the stimulus for fleetd agent to reconcile what needs to be done locally, then what stimulus remains after --disable_watches is enabled? You say that latency increases, but what stimuli remain? Does fleetd also put the agent under some timer? Thanks.

**Comment by jonboulle** [comment]
There is a periodic timer that wakes up the agent every 5 seconds to do
this reconciliation.

**Pull request merged**